src/common/components/Instructors.jsx

```diff
@@ -8,10 +8,18 @@ import { adjustNumberBeetwenMinMax } from '../../utils';
 import Heading from './Heading';
 import Icon from './Icon';
 
+function orderArrayByRole(array) {
+  return array.sort((a, b) => {
+    if (a.role === 'TEACHER') return -1;
+    if (b.role === 'TEACHER') return 1;
+    return 0;
+  });
+}
+
 function Instructors({ isLoading, list, limit, ...rest }) {
   const { t } = useTranslation('common');
```

Comment:
The ideal solution would be to order the list once, store the ordered list in a variable and use it later